netcloud/scripts/README.md

```diff
@@ -0,0 +1,23 @@
+## mitmproxy_netcloud_logging.py
+
+Script for parsing and logging NetCloud packets with mitmproxy (see [here](../README.md#mitm-netcloud-traffic-with-mitmproxy) for details).
+
+## parse_netcloud_packets_from_yaml.py
+
+Parses NetCloud packets from a YAML file exported by Wireshark (see [here](../README.md#decrypt-netcloud-traffic-in-wireshark) for details).
+
+## test_netcloud_registration.py
+
+Script for poking at the router registration process with NetCloud.
+
+Quick instructions:
+
+- Install the `netifaces` package: `$ pip install netifaces`
+- The script requires the `api` folder and the `stream.pyc` file
+  - Note: The `stream.pyc` file has been patched to work on a Mac. You may decompile it with [decompyle3](https://github.com/rocky/python-decompile3) and adapt it to your needs.
+- In the script, change the variables which have `Required` in the comment
+- Adjust the flags as needed to control the different registration stages. **Note**:
+  - `_INSECURE_ACTIVATION` only works for *new* devices that haven't been already registered with NetCloud.
```

Comment:
actually it **worked**, this has been patched by Cradlepoint.

Reply:
Removed.